level-3/README.md

```diff
@@ -0,0 +1,25 @@
+# karma-clone
+## Level 3
+
+Now let's make this website responsive! 
+
+### Set up
+
+You can choose to add your code for Level 3 to the same branch you were using for the previous levels, or to create a new 'level-3' branch. 
+
+# Instructions
+
+- Use the techniques you've learned in class to make both the homepage and the Order page look good on mobile. 
+- You'll find the mobile designs inside the 'Level 3' folder. 
```

Comment:
```suggestion
```
Same as my other suggestion, WDYT about removing this as the designs are already shown at the bottom of this readme?

Reply:
Again, I don't think it matters. It means students know that they can use the file in the folder as a reference - also easier to zoom in on than in the readme.

Reply:
No problem. Could we update `'Level 3'` to `` `level-3` `` to match the folder name?